### DISCUSSION TITLE
What is prysmctl --num-validators option for testnet generate-genesis.

### DISCUSSION THREAD

**Discussion by KP-Universe** [OffchainLabs/prysm#13663] [General]
# 💎 Issue
I'm just trying this example that you provide:
https://docs.prylabs.network/docs/advanced/proof-of-stake-devnet

when I use command "./prysmctl testnet generate-genesis --fork capella --num-validators 64"
Can't understand --num-validators option description... "Number of validators to deterministically generate in the genesis state (default: 0)"

I am not sure that means "There will be 64 validators in the genesis state" or "there is a maximum of 64 validators" or "there is a minimum of 64 validators".
I would like to know if setting the value to 64 allows for more validators to join, or if it requires a minimum of 64 validators to be present.

And 1 more question, I think the node(we need to run at first for genesis) didn't stake to run validators. I checked the node's account it's still has same balance before running validators.
If the settings for testnet doesn't need a deposit for staking, how to edit that settings to "need to stake 32eth for running 1 validator"?
### Background

 Context and background information on the discussion...

**Comment by nisdas** [comment]
@KP-Universe That flag just specifies how many validators there will be in the genesis state.
